FreshRSS/FreshRSS
# Improved: Sharing/Integration configuration

Ref. #3812 

Changes proposed in this pull request:

- improved UI of sharing configurations
- added: sharing service can flagged as deprecated
- flagged "Blogotext" as depricated
- added: documentation about sharing services

Before:
![grafik](https://user-images.githubusercontent.com/1645099/158058845-867f781c-f535-482d-a849-d261c75591e6.png)

and:

![grafik](https://user-images.githubusercontent.com/1645099/158058866-64f33827-0f6c-4707-8c19-bf0f20fc82a2.png)



After:
![grafik](https://user-images.githubusercontent.com/1645099/158058701-4c98437f-1122-46d7-8780-10d2e33ce566.png)

and:

![grafik](https://user-images.githubusercontent.com/1645099/158058720-3cfd95b7-434a-4bae-a653-2dda6127333b.png)


How to test the feature manually:

Configuration:
1. go to configuration => sharing
2. add/delete/change the sharing services
3. add Blogotext as sharing service (as URL you could use http://example.net)
4. drag&drop the services

Sharing
1. go to normal view and open an article
2. open the sharing dropdown menu
3. see the red sign next to Blogotext

Pull request checklist:


What left to do in another PR?

How to delete a sharing service without errors
